tests/frametest.c

```diff
@@ -658,6 +658,28 @@ int basicTests(U32 seed, double compressibility)
         CHECK( LZ4F_freeCompressionContext(cctx) ); cctx = NULL;
     }
 
+    DISPLAYLEVEL(3, "getBlockSize test: \n");
+    { size_t result;
+      for (unsigned blockSizeID = 4; blockSizeID < 8; ++blockSizeID) {
```

Comment:
unfortunately, this is a `C99` construct.
We try to preserve `C90` compatibility in most `lz4` codebase, 
including its tests tools.

You'll have to declare `unsigned blockSizeID;` just before the `for(`.